superset/assets/spec/javascripts/explore/components/MetricsControl_spec.jsx

```diff
@@ -224,7 +224,7 @@ describe('MetricsControl', () => {
 
       expect(!!wrapper.instance().selectFilterOption(
         { type: 'VARCHAR(255)', column_name: 'source', optionName: '_col_source' },
-        'Sou',
+        'sou',
```

Comment:
FYI react-select always passes lower case strings so I removed the toLowerCase call.

Reply:
TIL!